Sources/HCICreateConnection.swift

```diff
@@ -0,0 +1,190 @@
+//
+//  HCICreateConnection.swift
+//  Bluetooth
+//
+//  Created by Carlos Duclos on 7/26/18.
+//  Copyright © 2018 PureSwift. All rights reserved.
+//
+// swiftlint:disable function_parameter_count
+
+import Foundation
+
+// MARK: - Method
+
+public extension BluetoothHostControllerInterface {
+    
+    /// Create Connection Command
+    ///
+    /// This command causes the Link Manager to create a connection to the remote device with the BD_ADDR specified by the command parameters. This command causes the local BR/EDR Controller to begin the Page process to create a link level connection. The Link Manager will determine how the new ACL connection is established. This ACL connection is determined by the current state of the device, its piconet, and the state of the device to be connected.
+    func linkControlCreateConnection(address: Address,
```

Comment:
I don't think the HCI Command Opcode Group prefix (`linkControl`) is necessary, only Low Energy has its own separate category.